addons/website_blog/views/website_blog_templates.xml

```diff
@@ -264,9 +264,9 @@ list of filtered posts (by date or tag).
             </div>
 
             <!-- No-Sidebar Layout -->
-            <div t-if="not opt_blog_post_sidebar" t-attf-class="mx-auto #{opt_blog_post_readable and 'o_wblog_read'}">
+            <div t-if="not opt_blog_post_sidebar" t-attf-class="#{opt_blog_post_readable and 'o_container_small'}">
                 <div class="d-flex flex-column flex-lg-row">
-                    <div id="o_wblog_post_content" t-attf-class=" #{opt_blog_post_readable and 'o_wblog_read w-100 flex-shrink-0'}">
+                    <div id="o_wblog_post_content" t-attf-class=" #{opt_blog_post_readable and 'o_container_small w-100 flex-shrink-0'}">
```

Comment:
I guess we can fix those container inside container later. Also, it is in fact a container inside a container inside a container... that makes our container_small appear 30px wider in blogs :/ We'll fix that later.